docs/submitting-clinical-data.md

```diff
@@ -2,15 +2,28 @@
 id: submitting-clinical-data
 title: Submitting Clinical Data
 ---
+Clinical data consists of all supporting data about your programs donors and their health data.
 
-## Uploading Data
+> Note: Before you can submit clinical data, your data must be assigned ARGO Identifiers through the sample registration process. See [Registering Samples](/s) for instructions on how to register samples.
 
-### How To
 
-### Links
+## How To Submit Clinical Data
+1. Navigate to the **Submission** area in the top menu.
+1. Click on the **Clinical Submission** section in the left menu pane for your program.
+1. Download a **TSV Template** for the data type that you are submitting. Format your file(s) according to the [Data Dictionary](/dictionary) specifications for each clinical data type.
+1. Once your file(s) are formatted, click **Upload Files** and select the file. Only TSV file types are supported, and the file names must begin with data type that is being submitted.  For example, to _donor-\<anything\>.tsv_ would be a correctly formatted file name.
+1. If there are any errors in your file, they will be displayed on the Submission UI. The error report will also be available for download.
+1. Valid files will be available for preview on the UI. You can browse through the file contents to review.
 
 ## Completing Your Data Submission
 
 ### Validating Submission
+Submitted data must be reviewed for accuracy, against both newly submitted data and data that has previously been submitted.
+
+1. Once you are ready, click the **Validate** button.  Your entire submission will be checked for data consistency.
+1. If there are any errors in your submission, hey will be displayed on the Submission File Preview area. The error report will also be available for download.  Correct the errors in your source files and reupload the files.
```

Comment:
Upon validating, if there are any errors in your submission, they will be displayed within the submission workspace. The error report will also be available for download. You must fix all of the errors that are listed within your files file and then reupload them.